test/acceptance/test/utils.go

```diff
@@ -30,15 +30,16 @@ import (
 
 const THIRTY_SECOND_TIMEOUT time.Duration = 30 * time.Second
 const EVENTUALLY_DEFAULT_TIMEOUT time.Duration = 60 * time.Second
-const TIMEOUT_TWO_MINUTES time.Duration = 120 * time.Second
+const TIMEOUT_FIVE_MINUTES time.Duration = 5 * time.Minute
 const INSTALL_RESET_TIMEOUT time.Duration = 300 * time.Second
 const NAMESPACE_TERMINATE_TIMEOUT time.Duration = 600 * time.Second
-const INSTALL_PODS_READY_TIMEOUT time.Duration = 3 * time.Minute
+const INSTALL_PODS_READY_TIMEOUT time.Duration = 6 * time.Minute
 const WEGO_DEFAULT_NAMESPACE = wego.DefaultNamespace
 const WEGO_UI_URL = "http://localhost:9001"
 const SELENIUM_SERVICE_URL = "http://localhost:4444/wd/hub"
 const SCREENSHOTS_DIR string = "screenshots/"
 const DEFAULT_BRANCH_NAME = "main"
+const CI = "CI"
```

Comment:
This is more out of my curiosity. Why are you setting this value and then checking the os.getenv for it.

Reply:
This is used in quite a few places search for `os.Getenv(CI)`